roles/docker/meta/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 galaxy_info:
   author: your name
```

Comment:
Silly Atom editor re-writing files for whitespace. #949 in to fixup roles/docker/meta/main.yml separate from this PR